app/models/user.rb

```diff
@@ -490,14 +492,24 @@ def senior_delegate_for?(user)
     user.senior_delegates.include?(self)
   end
 
-  def banned?
-    group_member?(UserGroup.banned_competitors.first)
+  def age_in_years
+    age = Date.today.year - dob.year
```

Comment:
`dob` is not necessarily set. Users who "just" sign up without ever having competed before are not required to enter their DOB.

This value is only "backfilled" as part of the "Claim WCA ID" process. Note that users can claim a WCA ID directly while signing up/creating their account, but that doesn't change the fact that this field is only filled if a WCA ID has been claimed.

Reply:
Hmm, interesting. Obviously I need to add `nil` support, but more deeply we need to get a clear set of requirements from WCT on how to handle this - requested via email